GOVERNANCE.md

```diff
@@ -0,0 +1,191 @@
+    * Have a voice, but not a vote, in Core Maintainer decision-making meetings
+
+#### Process of becoming a maintainer:
+1. A current reviewer must be sponsored by a Maintainer of the repository in question or a Core Maintainer. The Maintainer or Core Maintainer will open a PR against the repository and add the nominee as a Maintainer in the MAINTAINERS.md file. The need for a sponsor is removed for Emeritus Maintainers, who may open this pull request themselves.
+2. The nominee will add a comment to the PR confirming that they agree to all requirements and responsibilities of becoming a Maintainer.
+3. A majority of the current Maintainers of the repository or subproject (including Core Maintainers) must then approve the PR. The need for a majority is removed for Emeritus Maintainers, who require only 2 current Maintainers or Core Maintainers to approve their return.
+
+### Core Maintainer
+Description: As the Podman project is composed of a number of subprojects, most maintainers will not have full knowledge of the full project and all its technical aspects. Those that do are eligible to become Core Maintainers, responsible for decisions affecting the entire project. Core Maintainers may act as a maintainer in all repositories and subprojects of the Podman Project.
+
+* Responsibilities include:
+    * All responsibilities of a maintainer, but applied to all repositories that make up the project
+    * Determining strategy and policy for the project
+* Requirements
+    * Experience as a Maintainer for at least 3 months
+    * Demonstrates a broad knowledge of all components, repositories, and subprojects of the Podman project.
+    * Is able to exercise judgment for the good of the project, independent of their employer, friends, or team
+    * Mentors new Maintainers and Core Maintainers
+    * Able to make decisions and contributions affecting the whole project, including multiple subprojects and repositories
+    * Can commit to maintaining a high level of contribution to the project as a whole
+* Additional privileges:
+    * Represent the project in public as a senior project member
+	* Represent the project in interactions with the CNCF
+    * Have a vote in Core Maintainer decision-making meetings
+
+#### Process of becoming a Core Maintainer:
+1. A current maintainer must be sponsored by Core Maintainer. The Core Maintainer will open a PR against the main Podman repository and add the nominee as a Core Maintainer in the MAINTAINERS.md file.
+2. The nominee will add a comment to the PR confirming that they agree to all requirements and responsibilities of becoming a Core Maintainer.
+3. A majority of the current Core Maintainers must then approve the PR.
+4. If, for some reason, all existing members are inactive according to the Inactivity policy below or there are no Core Maintainers due to resignations, a supermajority (66%) vote of maintainers can bypass this process and approve new Core Maintainers directly.
```

Comment:
This section seems to be for Podman only, while the rest of the document seems to be for use by multiple projects.

Reply:
I'll take that back, I see what's happening now.

Reply:
We need one repo to be the "main repo" for purposes of the project as a whole. I'm using Podman as that given that's what we were accepted under. Podman will have the definitive source of truth for multi-repo roles